Bottom: d5e1f5c022a7bd64520d957debca58fabfdbfe1a
Top:    67024f81d01ffab43c8ad6a4b5599a6f79e33feb
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-12 01:25:59 +0000

Make --enable-gl default when building the GTK HID


---

diff --git a/configure.ac b/configure.ac
index 80cb6db..26b20cb 100644
--- a/configure.ac
+++ b/configure.ac
@@ -386,8 +386,12 @@ fi
 AC_MSG_CHECKING([for whether to use GL drawing])
 AC_ARG_ENABLE([gl],
 [  --enable-gl           Enable GL drawing (with GTK HID)],
-[],[enable_gl=no])
-
+[],[
+	case " $with_gui " in
+		*\ gtk\ *) enable_gl=yes;;
+		* ) enable_gl=no;;
+	esac
+])
 AC_MSG_RESULT([$enable_gl])
 AM_CONDITIONAL(USE_GL, test x$enable_gl = xyes)
